src/components/program/ProgramContentMenu.tsx

```diff
@@ -279,6 +294,44 @@ const SortBySectionItem: React.VFC<{
     programContentId?: string
   }>()
   const [previousPage] = useQueryParam('back', StringParam)
+  const [exerciseId] = useQueryParam('exerciseId', StringParam)
+
+  const type = contentCurrentProgress?.programContentType || ''
+  const { loadingSpecificExercise, specificExercise, refetchSpecificExercise, isTaken, exerciseAnswerer } =
+    useSpecificExercise(programContent.id, currentMemberId || '', exerciseId)
+  const {
+    loadingExamId,
+    errorExamId,
+    loadingExam,
+    errorExam,
+    exam,
+    refetch: refetchExam,
+  } = useExam(programContent.id, specificExercise)
+
+  const questionsIdCondition = exam.questions.map(question => {
+    return question.id
+  })
+  const { currentExamData, gaindedPointsTotal, refetchCurrentExamData, questionPointsTotal } =
```

Comment:
choose a method to process
1. add loading and error status
2. remove variable which is declared but its value is never read